hardware_interface/src/resource_manager.cpp

```diff
@@ -545,6 +551,8 @@ class ResourceStorage
 
   std::unordered_map<std::string, HardwareComponentInfo> hardware_info_map_;
 
+  std::unordered_map<std::string, std::vector<std::string>> hardware_used_by_controllers_;
```

Comment:
could we please document in a comment above these variables what mapping they provide? the variable names are not great -> perhaps that could be improved too

Reply:
done